docs/src/pages/components/HorizontalNav/AndroidHorizontalNav.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Horizontal nav
+platform: android
+documentationId: net.skyscanner.backpack.horisontalnav
```

Comment:
As I can't put comments inside the frontmatter this is taken from the js old file

>     This is deliberately misspelled because the package name
>     was published with a typo. :seenoevil: